src/app/feed/feed-card/feed-card.component.ts

```diff
@@ -131,31 +135,32 @@ export class FeedCardComponent implements OnInit {
 		}
 	}
 
-	private sanitizeAndEmbedURLs(links) {
-		links.forEach((link, i) => {
-			const videoid = links[i].match(/(?:https?:\/{2})?(?:w{3}\.)?youtu(?:be)?\.(?:com|be)(?:\/watch\?v=|\/)([^\s&]+)/);
+	private sanitizeVideoURLs() {
+		this.feedItem.videos.forEach((video) => {
+			const videoid = video.match(/(?:https?:\/{2})?(?:w{3}\.)?youtu(?:be)?\.(?:com|be)(?:\/watch\?v=|\/)([^\s&]+)/);
```

Comment:
Yes, thanks @vibhcool for pointing this out, though the videos are mostly from youtube, there are other providers as well. I will look into that, for now I just want to make the videos to work again in the application. I have created the issue [#415](https://github.com/fossasia/loklak_sear)h/issues/415) for the same and will look to make this better. :)